mariadb/stack.yml

```diff
@@ -1,16 +1,17 @@
 # Use root/example as user/password credentials
-version: '3.1'
```

Comment:
If we drop this, we also need to drop the references to `docker stack deploy`; see also https://github.com/docker-library/docs/pull/2516#issuecomment-2574035913

Reply:
> If we drop this, we also need to drop the references to `docker stack deploy`; see also [#2516 (comment)](https://github.com/docker-library/docs/pull/2516#issuecomment-2574035913)

https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete

Reply:
more explicitly:

https://docs.docker.com/engine/swarm/stack-deploy "The docker stack deploy command uses the legacy [Compose file version 3](https://docs.docker.com/reference/compose-file/legacy-versions/) format, used by Compose V1"

[https://docs.docker.com/compose/intro/history/ "](https://docs.docker.com/compose/intro/history/#introduction) (from image - Compose V1 / Compose file version  - "Used version top level element". So if you remove version, remove the `docker stack deploy` like @tianon  said.